### PR TITLE
Error messages and success toasts

### LINKED ISSUE #190: Check warning/success messages
Replace `@if ($errors->any())` with concrete conditionals, so the warnings are only shown where they  should.

### PR DESCRIPTION
**Pull request by kdmnk**
Fixes #190

### CODE REVIEW

**Comment by kdmnk** [comment]
Didn't test everything so pay attention to typos

File: app/Http/Controllers/InternetController.php [hunk 1]
```diff
@@ -156,7 +160,7 @@ public function addMacAddress(Request $request)
 
         $this->autoApproveMacAddresses(Auth::user());
 
-        return redirect()->back();
+        return redirect()->back()->with('message', __('general.succesfully_added'));
```
Comment (luksan47) [inline app/Http/Controllers/InternetController.php:163]: nit: successfully

File: app/Http/Controllers/PrintController.php [hunk 2]
```diff
@@ -118,7 +122,7 @@ public function transferBalance(Request $request) {
         $from_account->decrement('balance', $balance);
         $to_account->increment('balance', $balance);
 
-        return redirect()->route('print');
+        return redirect()->back()->with('message', __('general.succesful_transaction'));
```
Comment (luksan47) [inline app/Http/Controllers/PrintController.php:125]: nit: successful

File: app/Http/Controllers/PrintController.php [hunk 3]
```diff
@@ -136,7 +140,8 @@ public function modifyBalance(Request $request) {
         }
         $print_account->update(['last_modified_by' => Auth::user()->id]);
         $print_account->increment('balance', $balance);
-        return redirect()->back();
+
+        return redirect()->back()->with('message', __('general.succesful_modification'));
```
Comment (luksan47) [inline app/Http/Controllers/PrintController.php:144]: nit: successful

File: app/Http/Controllers/PrintController.php [hunk 4]
```diff
@@ -157,7 +162,7 @@ public function addFreePages(Request $request) {
             'comment' => $request->comment,
         ]);
 
-        return redirect()->back();
+        return redirect()->back()->with('message', __('general.succesfully_added'));
```
Comment (luksan47) [inline app/Http/Controllers/PrintController.php:165]: nit: successful

File: app/Http/Controllers/PrintController.php [hunk 5]
```diff
@@ -279,10 +284,9 @@ private function printFile($file, $cost, $is_two_sided, $number_of_copies) {
     }
 
     private function handleNoBalance($validator) {
-        $validator->errors()->add('balance', __('print.no_balance'));
-        return back()
-            ->withErrors($validator)
-            ->withInput();
+
+        return back()->with('error',  __('print.no_balance'));
```
Comment (luksan47) [inline app/Http/Controllers/PrintController.php:288]: withInput returns the input as well, I think we should use this.

**Pull request merged**